how-to/register-with-platform-windows/public/manifest.fin.json

```diff
@@ -30,7 +30,19 @@
 						"protocols": ["mailto"]
 					}
 				}
-			}
+			},
+			"preloadScripts": [
+				{
+					"url": "http://localhost:8080/common/style/style-changed-preload.js"
```

Comment:
If the theme change is currently only triggered by browser would there be a need for this preload script? Nothing would trigger it here. Maybe exclude it from this sample so as to not overcomplicate it (as this platform doesn't do theming).

Reply:
This was just pasted in to the wrong file, removed